public/language/en-GB/admin/settings/uploads.json

```diff
@@ -21,6 +21,10 @@
 	"topic-thumb-size": "Topic Thumb Size",
 	"allowed-file-extensions": "Allowed File Extensions",
 	"allowed-file-extensions-help": "Enter comma-separated list of file extensions here (e.g. <code>pdf,xls,doc</code>). An empty list means all extensions are allowed.",
+	"upload-limit-threshold": "Rate limit user uploads to:",
+	"upload-limit-threshold-per": "per",
```

Comment:
This is not easily internationalisable, different languages may not use "per" in this manner.

Use `%1 per minute` and %1 per minutes` instead.

Reply:
this use case works and can be translated, see template